crates/cxx-qt-build/src/lib.rs

```diff
@@ -688,6 +693,15 @@ impl CxxQtBuilder {
         for include_path in include_paths {
             builder.include(include_path);
         }
+
+        if !Self::is_debug() {
```

Comment:
should we only do this on Windows ? Or do we also need to on other platforms :thinking:

Reply:
Seems when building a plugin you use `if cc_builder.get_compiler().is_like_msvc() {` to set this.

Does this *also* need to be enabled in all cc builders too then ?